src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/python/schema_validation.py

```diff
@@ -333,7 +343,10 @@
 TARGET_SCHEMA = {
     Optional("path"): Or(str, int, TARGET_LIST_SCHEMA),
     Optional("tags"): {
-        And(str, Regex(r"\A.{1,128}\Z")): And(str, Regex(r"\A.{0,256}\Z"))
+        And(str, Regex(r"\A.{1,128}\Z")): Or(
+            And(str, Regex(r"\A.{0,256}\Z")),
+            And(list)
+        )
```

Comment:
I don't think this change is related to this PR. Can you move that to its own PR if we need to get that changed?

Reply:
@AndyEfaa Could you move this change to its own PR?
Plus I think it would be good to validate the list of tags too. So instead of the `And(list)` it should validate each item in the list too.